src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditDetails.tsx

```diff
@@ -41,7 +45,28 @@ const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({
             />
           </FormControl>
         </Grid>
-        <Grid item xs={4}>
+        <Grid item lg={2.4} xs={12}>
+          <FormControl fullWidth>
+            <ChangeRequestDropdown control={control} name="crId" errors={errors} />
```

Comment:
@RChandler234 Should the TeamsDropdown component also take in errors as a prop and display them if required?

Reply:
yes